jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractListPresenter.java

```diff
@@ -102,4 +105,11 @@ public void refreshGrid() {
                                                                     true);
         }
     }
+
+    public void onDeselectAllItemsEvent(@Observes DeselectAllItemsEvent event) {
+        if (getListView().getListGrid() != null) {
+            getListView().getListGrid().deselectAllItems();
+        }
+        refreshGrid();
```

Comment:
just not sure if we always want to refresh the grid when deselectall is executed.